recipe/build.sh

```diff
@@ -2,38 +2,4 @@
 
 set -e
 
-if [[ "$c_compiler" == "gcc" ]]; then
-  export PATH="${PATH}:${BUILD_PREFIX}/${HOST}/sysroot/usr/lib"
-fi
-
-if [[ $(uname) == Darwin ]]; then
-  export LIBRARY_SEARCH_VAR=DYLD_FALLBACK_LIBRARY_PATH
-elif [[ $(uname) == Linux ]]; then
-  export LIBRARY_SEARCH_VAR=LD_LIBRARY_PATH
-fi
-
-export PYTHON="$PYTHON"
-export PYTHON_LDFLAGS="$PREFIX/lib"
-export LDFLAGS="$LDFLAGS -L$PREFIX/lib -Wl,-rpath,$PREFIX/lib"
-export CFLAGS="$CFLAGS -fPIC -I$PREFIX/include"
-
-mkdir ../build && cd ../build
-cmake -D CMAKE_INSTALL_PREFIX=$PREFIX \
-      -D ENABLE_JPG=1 \
-      -D ENABLE_NETCDF=1 \
-      -D ENABLE_PNG=1 \
-      -D ENABLE_PYTHON=1 \
-      -D ENABLE_FORTRAN=0 \
-      -D ENABLE_AEC=1 \
-      $SRC_DIR
-
-make -j $CPU_COUNT
-
-export ECCODES_TEST_VERBOSE_OUTPUT=1
-eval ${LIBRARY_SEARCH_VAR}=$PREFIX/lib
-
-if [[ $(uname) == Linux ]]; then
-  ctest -j $CPU_COUNT
-fi
-
-make install
+${PYTHON} -m pip install . -vv --no-deps
```

Comment:
@beckermr Thanks for your efforts! Could you have a look at this https://pypi.org/project/eccodes-python/ under `Fast Bindings`. I'm a bit confused by this paragraph. Does the above line compile the CFFI stuff or just install the slow pythin ABI?